packages/account/src/Components/form-field-info/__tests__/form-field.info.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { FormFieldInfo } from '../form-field-info';
+
+describe('FormFieldInfo', () => {
+    it('should render the component', () => {
+        render(<FormFieldInfo message='Info content' />);
+        const popover = screen.getByTestId('dt_form-field-info__popover');
+        expect(popover).toBeInTheDocument();
+    });
+
+    it('should toggle popover on click', () => {
+        render(<FormFieldInfo message='Info content' />);
+        const message = screen.queryByText('Info content');
+        expect(message).not.toBeInTheDocument();
+
+        const popover = screen.getByTestId('dt_form-field-info__popover');
+
+        fireEvent.click(popover);
```

Comment:
Currently we mostly use userEvent in testcases, you can try it here as well.
just a suggestion, you can ignore it

Reply:
Thanks @yauheni-deriv 
Update.

Reply:
Are u planning to use userEvent?